src/Sweep.cpp

```diff
@@ -1367,15 +1367,15 @@ void DigitalContactTracingSweep(double t)
 					else if (P.FindContactsOfDCTContacts)
 					{
 						//check every day to see if contacts become index cases - but they have to be infectious. Otherwise we could set the trigger time and cause their contacts to be traced when they are not being traced themselves.
-						if ((Hosts[contact].index_case_dct == 0) && ((abs(Hosts[contact].inf) == 2) || (Hosts[contact].inf == -1)))
+						if ((Hosts[contact].index_case_dct == 0) && ((abs((int) Hosts[contact].inf) == 2) || (Hosts[contact].inf == InfStat::InfectiousAlmostSymptomatic)))
```

Comment:
Replace the "2" with InfStat::InfectiousAsymptomaticNotCase

Reply:
But `Case` also gives abs value 2, as it was intended previously. Are you sure it's supposed to be replaced with that?

Reply:
Syntactially correct - abs() would result in InfStat::InfectiousAsymptomaticNotCase - but again see earlier comment - it may be clearer to expand these and remove the necessity for abs